### PR TITLE
Add release for EC v0.1 alpha

### PR DESCRIPTION
**Pull request by simonbaird**
Note that I'm using the rhtas-tenant as an example rather than the example from the documentation. Notably the target is rhtap-releng-tenant and not rh-managed-rhtap-ser-tenant. (Let me know if I'm copying an incorrect example.)

### CODE REVIEW

**Review by zregvart — APPROVED** [review]

File: auto-generated/cluster/stone-prd-rh01/tenants/rhtap-contract-tenant/appstudio.redhat.com_v1alpha1_releaseplan_ec-v01-alpha-registry-redhat-io.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+metadata:
+  labels:
+    release.appstudio.openshift.io/auto-release: "true"
+    release.appstudio.openshift.io/standing-attribution: "true"
```
Comment (zregvart) [inline auto-generated/cluster/stone-prd-rh01/tenants/rhtap-contract-tenant/appstudio.redhat.com_v1alpha1_releaseplan_ec-v01-alpha-registry-redhat-io.yaml:6]: Do we need `release.appstudio.openshift.io/author` here?

Comment (simonbaird) [reply auto-generated/cluster/stone-prd-rh01/tenants/rhtap-contract-tenant/appstudio.redhat.com_v1alpha1_releaseplan_ec-v01-alpha-registry-redhat-io.yaml:6]: I don't see it on other release plans, so I guess no..?

Comment (simonbaird) [reply auto-generated/cluster/stone-prd-rh01/tenants/rhtap-contract-tenant/appstudio.redhat.com_v1alpha1_releaseplan_ec-v01-alpha-registry-redhat-io.yaml:6]: See also [these docs](https://docs.google.com/document/d/1nCHMYxyJQW_fAH8z97uk16IvYTQaodbUn87fMGgw8YA/edit#heading=h.4p8list4bfyj).

**Comment by simonbaird** [comment]
@scoheb does it look good?

File: cluster/stone-prd-rh01/tenants/rhtap-contract-tenant/release_plan.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+metadata:
+  labels:
+    release.appstudio.openshift.io/auto-release: 'true'
+    release.appstudio.openshift.io/standing-attribution: 'true'
+  namespace: rhtap-contract-tenant
+  name: ec-v01-alpha-registry-redhat-io
+spec:
+  application: ec-v01-alpha
+  target: rhtap-releng-tenant
+  displayName: ec-v0.1-alpha release
```
[the remark this answers is not in the document]
Comment (simonbaird) [reply cluster/stone-prd-rh01/tenants/rhtap-contract-tenant/release_plan.yaml]: Okay, done.

**Review by scoheb — APPROVED** [review]

**Pull request merged**